Code/Mantid/Build/CMake/FindPoco.cmake

```diff
@@ -61,13 +61,25 @@ if( POCO_INCLUDE_DIR )
   file ( STRINGS ${VERSION_FILE} POCO_VERSION REGEX ${VERS_REGEX} )
   # pull out just the part after the 0x
   string( REGEX REPLACE ${VERS_REGEX} "\\1" POCO_VERSION ${POCO_VERSION} )
+
   # Pretty format
   string( SUBSTRING ${POCO_VERSION} 0 2 POCO_VERSION_MAJOR )
   string( REGEX REPLACE "^0" "" POCO_VERSION_MAJOR ${POCO_VERSION_MAJOR} )
+  if ("${POCO_VERSION_MAJOR}" STREQUAL "")
+    set( POCO_VERSION_MAJOR "0")
+  endif()
+
   string( SUBSTRING ${POCO_VERSION} 2 2 POCO_VERSION_MINOR )
   string( REGEX REPLACE "^0" "" POCO_VERSION_MINOR ${POCO_VERSION_MINOR} )
+  if ("${POCO_VERSION_MINOR}" STREQUAL "")
+    set( POCO_VERSION_MINOR "0")
+  endif()
+
   string( SUBSTRING ${POCO_VERSION} 4 2 POCO_VERSION_PATCH )
   string( REGEX REPLACE "^0" "" POCO_VERSION_PATCH ${POCO_VERSION_PATCH} )
```

Comment:
Thank you! I just updated the branch to include this line in place of the if statements.